pallets/proposals/src/impls.rs

```diff
@@ -553,30 +564,41 @@ impl<T: Config> Pallet<T> {
                     }
                 });
 
-        let available_funds: BalanceOf<T> = unlocked_funds - project.withdrawn_funds;
+        let withdrawable: BalanceOf<T> = unlocked_funds - project.withdrawn_funds;
```

Comment:
seem to be missing some safe math here, my fault
Can this be saturating_sub pls